crates/astria-sequencer/src/bridge/storage/values/transaction_id.rs

```diff
@@ -0,0 +1,63 @@
+use std::{
+    borrow::Cow,
+    fmt::{
+        self,
+        Display,
+        Formatter,
+    },
+};
+
+use astria_core::primitive::v1::{
+    TransactionId as DomainTransactionId,
+    TRANSACTION_ID_LEN,
+};
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+use telemetry::display::hex;
+
+use super::{
+    Value,
+    ValueImpl,
+};
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(in crate::bridge) struct TransactionId<'a>(Cow<'a, [u8; TRANSACTION_ID_LEN]>);
+
+impl<'a> Display for TransactionId<'a> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        hex(self.0.as_slice()).fmt(f)
```

Comment:
We should assert that this matches the formatting of `TransactionId`.

Reply:
As discussed offline, we can skip that since we're using Debug formatting.